packages/web-components/src/components/card-section-simple/card-section-simple.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+import styles from './card-section-simple.scss';
+import DDSContentSection from '../content-section/content-section';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The Card Section pattern
+ *
+ * @element dds-card-section-simple
+ */
+@customElement(`${ddsPrefix}-card-section-simple`)
+class DDSCardSectionSimple extends StableSelectorMixin(DDSContentSection) {
+  /**
+   * Applies section attribute
+   */
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'section');
+    }
+    super.connectedCallback();
+  }
+
+  render() {
+    return html`
+      ${super.render()}
+    `;
+  }
```

Comment:
If `super.render()` is the only thing, we don't have to override `render()`:

```suggestion
```

Reply:
Makes sense, thanks!